src/test/java/ca/corefacility/bioinformatics/irida/NullReplacementDatasetLoader.java

```diff
@@ -0,0 +1,30 @@
+package ca.corefacility.bioinformatics.irida;
+
+import com.github.springtestdbunit.dataset.AbstractDataSetLoader;
+import org.dbunit.dataset.IDataSet;
+import org.dbunit.dataset.ReplacementDataSet;
+import org.dbunit.dataset.xml.FlatXmlDataSet;
+import org.dbunit.dataset.xml.FlatXmlDataSetBuilder;
+import org.springframework.core.io.Resource;
+
+import java.io.InputStream;
+
+/**
+ * Spring DBUnit loader which replaces '[null]' with an actual NULL value in the database.
+ */
+public class NullReplacementDatasetLoader extends AbstractDataSetLoader {
```

Comment:
Added this file because it's otherwise nearly impossible to insert a `NULL` with dbunit.  This uses a `ReplacementDataSet` to replace `[null]` in a dbunit xml file with an actual `NULL` in the object & database.